cheri-architecture.tex

```diff
@@ -53,7 +53,6 @@
   \\
   SRI International, University of Cambridge, and Arm Limited
 }
-\date{22nd September 2023}
```

Comment:
Maybe comment out in hopes of reminding us to re-add?

Reply:
Done